data/json/items/book/tailor.json

```
@@ -136,5 +136,25 @@
     "max_level": 6,
     "intelligence": 8,
     "time": "30 m"
+  },
+  {
+    "id": "scots_tailor",
+    "type": "BOOK",
+    "name": { "str": "Ye Scots Beuk o Tailoring", "str_pl": "copies of Ye Scots Beuk o Tailoring" },
```

```suggestion
    "name": { "str": "Ye Scots Beuk o Tailorin'", "str_pl": "copies of Ye Scots Beuk o Tailorin'" },
```
I refuse to believe the accent would spell it -ing 😛.